src/main/js/html.js

```diff
@@ -114,6 +129,20 @@
         rootcontainer.style.right = 0;
         rootcontainer.style.zIndex = 0;
 
+        if (options) { 
```

Comment:
Can this section of code move to below the definition of options inside context (setting `context.options.colorAdjust`) to avoid mutating the `options` object? 

The use case is that a library user has the same options they want to pass into the function on each call and after the first call the colorAdjust properties are mutated and can't be read by `imscUtils.parseColor`.

Reply:
Addressed in 9116902.